src/components/elements/input/RadioGroupInput.tsx

```diff
@@ -117,7 +117,6 @@ const RadioGroupInput = ({
               borderRadius: 1,
             },
           }),
-          ...sx,
```

Comment:
I checked usages of `RadioGroupInput` and found that it is only used in `DynamicField.tsx` and `YesNoRadio.tsx`, which in turn is only used in `DynamicField.tsx`, so I think this change is safe

Reply:
👍  this is better, anyway. I think if we have an outer `sx` prop it is most intuitive if it applies to the outermost component.